recipes/mistletoe-ebp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "mistletoe-ebp" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7f5e3a06c9dcd6a4afa328b1402c65c121c8ea2a98e30b397fc6ca4deacca7f1"
+
+build:
+  number: 0
+  entry_points:
+    - mistletoe = mistletoe.__main__:main
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - mistletoe
+  commands:
+    - mistletoe --help
+
+about:
+  home: "https://github.com/ExecutableBookProject/mistletoe-ebp"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "A fast, extensible Markdown parser in pure Python."
+
+extra:
+  recipe-maintainers:
+    - chrisjsewell
+    - choldgraf
```

Comment:
Can you just confirm that choldgraf is willing to be a co-maintainer?

Reply:
Yes I can confirm that @choldgraf is willing to be a co-maintainer

Reply:
thanks 👍